src/renderer/components/styles.ts

```diff
@@ -1,298 +0,0 @@
-// ==LICENSE-BEGIN==
-// Copyright 2017 European Digital Reading Lab. All rights reserved.
-// Licensed to the Readium Foundation under one or more contributor license agreements.
-// Use of this source code is governed by a BSD-style license
-// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
-// ==LICENSE-END==
-
-import { relative } from "path";
-
-const auto: "auto" = "auto";
-const fixed: "fixed" = "fixed";
-const hidden: "hidden" = "hidden";
-const absolute: "absolute" = "absolute";
-const column: "column" = "column";
-const wrap: "wrap" = "wrap";
-
-export const Styles = {
```

Comment:
Note that the removal of `src/renderer/components/styles.ts` partially addresses issue: https://github.com/readium/readium-desktop/issues/409
(we might want to file a separate issue to address "inline styles")